src/current/cockroachcloud/configure-scim-provisioning.md

```diff
@@ -128,27 +146,81 @@ To learn more, refer to [Deprovision a user](https://help.okta.com/en-us/Content
 
 A linked group that is unassigned from the app integration continues to appear in {{ site.data.products.db }} unless it is unlinked. Refer to [Automate Group Management](#automate-group-management).
 
-### Automate group management
+<a id="automate-group-management"></a>
+### Automate group management with Okta
 
-By default, users and groups are provisioned in {{ site.data.products.db }} only when they are assigned to the app integration, and changes to a group's membership in Okta are not automatically reflected in {{ site.data.products.db }}. When [Group Push](https://help.okta.com/en-us/Content/Topics/users-groups-profiles/usgp-about-group-push.htm) is enabled in an app integration, modifying group membership in Okta is automatically reflected in {{ site.data.products.db }}.
+The following sections show how to enable and manage Okta's Group Push feature. For more information and troubleshooting, refer to [Manage Group Push](https://help.okta.com/en-us/Content/Topics/users-groups-profiles/usgp-group-push-main.htm) in the Okta documentation.
 
-To enable Group Push and link groups:
+### Enable Group Push
+
+To enable Group Push and begin synchronizing a group with {{ site.data.products.db }}:
 
 1. Log in to Okta Admin Dashboard as an admin user.
 1. Click **Applications** and click the [SAML](configure-cloud-org-sso.html#saml) app integration for your {{ site.data.products.db }} organization.
 1. Click the **Push Groups** tab. Unless you disable **Push group memberships immediately**, changes you make in this tab will be applied immediately.
 
-   1. To link a group, click **Push Groups**, then select it.
+   1. To link a group:
+      1. Click **Push Groups**, then select either **Find groups by name** or **Find groups by rule**.
+      1. Find a group to push, and click its name.
+      1. Under **Match result & push action**, click **Create Group** to create a new group in {{ site.data.products.db }} and link the Okta group to it. If a group exists in {{ site.data.products.db }} with the same name, click **Link Group**.
    1. To unlink a linked group, select it, then click **Unlink pushed group**.
    1. If you disabled **Push group memberships immediately**, click **Push Now**.
 
-1. To disable Group Push, click **Deactivate Group Push**.
+### Disable Group Push
```

Comment:
This is part of the above Automate group management section, right? If yes, Should this have a #### header instead of ###?